dietpi/func/dietpi-banner

```diff
@@ -262,13 +262,7 @@ $GREEN_LINE"
 		# Let's Encrypt cert status
 		if (( ${aENABLED[16]} == 1 ))
 		then
-			local i certinfo='No certificate found'
-			for i in /etc/letsencrypt/live/*/cert.pem
-			do
-				[[ -f $i ]] || continue
-				certinfo=$(openssl x509 -enddate -noout -in "$i" | mawk '/notAfter=/{print "Valid until "$4"-"substr($1,10)"-"$2" "$3}' 2>&1)
-				break
-			done
+			local certinfo="$(sudo /boot/dietpi/func/dietpi-banner-certinfo)"
```

Comment:
```suggestion
			Obtain_Cert_Info()
			{
				local i
				for i in /etc/letsencrypt/live/*/cert.pem
				do
					[ -f "$i" ] || continue
					openssl x509 -enddate -noout -in "$i" | mawk '/notAfter=/{print "Valid until "$4"-"substr($1,10)"-"$2" "$3}' 2>&1
					return 0
				done
				echo 'No certificate found'
			}
			echo -en "$GREEN_BULLET ${aCOLOUR[1]}${aDESCRIPTION[16]} "
			if (( $UID ))
			then
				sudo dash -c "$(declare -f Obtain_Cert_Info); Obtain_Cert_Info"
			else
				Obtain_Cert_Info
			fi
```

Reply:
Oh, I did quite the same without seeing your suggestion ; I update with dash, and NOPASSWD control

Reply:
Actually,
Won't use dash

With dash [[ -e $i ]] won't work
If I use [ -e $i ], shellcheck won't accept : 
```
In ./dietpi/func/dietpi-banner line 227:
                [ -e $i ] || continue
                ^-------^ SC2292 (style): Prefer [[ ]] over [ ] for tests in Bash/Ksh.
                     ^-- SC2086 (info): Double quote to prevent globbing and word splitting.
Did you mean: 
                [[ -e "$i" ]] || continue
For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
  https://www.shellcheck.net/wiki/SC2292 -- Prefer [[ ]] over [ ] for tests i...
Error: Process completed with exit code 1.
```
=> I offer to use bash, to keep the script shellcheck-compliant

Reply:
Should be OK now, ready for review

Reply:
You can tell shellcheck to ignore issues. It is a help to find possible issues and help with consistent coding style, not something that should prevent us from more efficient code 😉.

I'll apply some refinement.